<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-04-20T19:42:17 IST" name="Default suite" finished-at="2022-04-20T19:42:37 IST" duration-ms="20151">
    <groups>
    </groups>
    <test started-at="2022-04-20T19:42:17 IST" name="Default test" finished-at="2022-04-20T19:42:37 IST" duration-ms="20151">
      <class name="com.inetbanking.testcases.LoginTC001">
        <test-method is-config="true" signature="init()[pri:0, instance:com.inetbanking.testcases.LoginTC001@564fabc8]" started-at="2022-04-20T19:42:15 IST" name="init" finished-at="2022-04-20T19:42:17 IST" duration-ms="1438" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method is-config="true" signature="setUp(java.lang.String)[pri:0, instance:com.inetbanking.testcases.LoginTC001@564fabc8]" started-at="2022-04-20T19:42:17 IST" name="setUp" finished-at="2022-04-20T19:42:30 IST" duration-ms="13425" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method signature="verifyLoginFuctionalityTC001()[pri:0, instance:com.inetbanking.testcases.LoginTC001@564fabc8]" started-at="2022-04-20T19:42:30 IST" name="verifyLoginFuctionalityTC001" finished-at="2022-04-20T19:42:36 IST" duration-ms="5352" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at com.inetbanking.pageobject.LoginPage.clickonLoginBtn(LoginPage.java:62)
at com.inetbanking.testcases.LoginTC001.verifyLoginFuctionalityTC001(LoginTC001.java:26)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginFuctionalityTC001 -->
        <test-method is-config="true" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.inetbanking.testcases.LoginTC001@564fabc8]" started-at="2022-04-20T19:42:36 IST" name="tearDown" finished-at="2022-04-20T19:42:37 IST" duration-ms="760" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginFuctionalityTC001 status=FAILURE method=LoginTC001.verifyLoginFuctionalityTC001()[pri:0, instance:com.inetbanking.testcases.LoginTC001@564fabc8] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="flushReport()[pri:0, instance:com.inetbanking.testcases.LoginTC001@564fabc8]" started-at="2022-04-20T19:42:37 IST" name="flushReport" finished-at="2022-04-20T19:42:37 IST" duration-ms="551" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flushReport -->
      </class> <!-- com.inetbanking.testcases.LoginTC001 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
